src/audio/kpb.c

```diff
@@ -135,6 +147,24 @@ static uint64_t kpb_task_deadline(void *data)
 	return SOF_TASK_DEADLINE_ALMOST_IDLE;
 }
 
+#if CONFIG_AMS
+
+/* KPD message*/
+DEFINE_UUID(ams_kpd_msg_uuid, AMS_KPD_MSG_UUID);
+
+static void kpb_ams_kpd_notification(const struct ams_message_payload *const ams_message_payload,
```

Comment:
naming nitpick: what is the meaning of the repeated "kpb" in the helper? is this intentional? Or can we just call this "kpb_ams_notification"?

Reply:
@plbossart it's not a repetition of `kpb`, it is another acronym `kpd` which stands for "key phrase detected" message.
But I agree it is a little bit confusing.

Reply:
oh wow, even on my 24' monitor I didn't see the kbp/kpd difference. Do you mind adding a comment so that this is not lost?